api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -150,6 +169,14 @@ public void testSyncComplianceTrainingStatusV2() throws Exception {
         .isEqualTo(Timestamp.from(Instant.ofEpochSecond(TIMESTAMP_SECS)));
     assertThat(user.getComplianceTrainingExpirationTime())
         .isEqualTo(Timestamp.from(Instant.ofEpochSecond(expiry)));
+    assertThat(
```

Comment:
This is a lot of duplicated code - what do you think about extracting a method we can call like `assertCompletion(RT_COMPLIANCE_TRAINING, TIMESTAMP_SECS)`?